db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Migrate/Email.class/__methods__/vmmigratetask_complete.rb

```diff
@@ -33,8 +33,8 @@
   end
 end
 
-if vm.nil?
-  vm = miq_task.vm unless miq_task.nil?
+if vm.nil? && miq_task
+  vm = $evm.vmdb('vm', miq_task.source_id)
```

Comment:
This can just reference `miq_task.source` which is a relationship exposed at the base miq_request_task service model.

Also, all the checking above for the vm object could be replaced with that call.  Lines 20 through 38 would be reduced to: `vm = miq_task.source`

cc @tinaafitz